src/main/java/micdoodle8/mods/galacticraft/api/spaceprojects/ISpaceProject.java

```diff
@@ -0,0 +1,51 @@
+package micdoodle8.mods.galacticraft.api.spaceprojects;
+
+import static micdoodle8.mods.galacticraft.api.spaceprojects.GCGlobalVariableStorage.*;
+
+import java.util.UUID;
+import net.minecraft.entity.player.EntityPlayer;
+
+public interface ISpaceProject {
```

Comment:
Is there a reason this is an interface? Having a default implemention for each method and also never overriding any kinda defeats the purpose.

Reply:
Its an interface so that certain blocks can implement it and use it. Its will be similar in nature to GT5u's wireless energy network. Just used for other things.

Reply:
I'm not a Java pro but to me this feels a bit "dirty". Having a class with static methods would achieve the same and you wouldn't need an extra class just to hold one or two variables.

Reply:
Yeah i was mostly copying over from the wireless eu network. Its good that now i know that there are better methods at least.